### DISCUSSION TITLE
pm2 and tt++

### DISCUSSION THREAD

**Discussion by DoubleUltera** [scandum/tintin#74] [Script help]
Hi everyone.

I'm trying to automate tt++ to run every boot using pm2 and load a .tin script but I encounter some errors. This might be something I need to ask in pm2 but thought I might as well ask here too.

I tried 

> :~/tt# pm2 start ./tt++

The error is as follows:

> 0|tt++     | tcgetattr: Inappropriate ioctl for device

Any tips would be appreciated. 

TIA

**Comment by scandum** [comment]
I assume this is an issue with pm2. I have no real idea what the problem might be.

**Comment by DoubleUltera** [comment]
Hi Scandum,

Thanks for taking a look into it. Appreciate everything.

**Reply by scandum** [reply]
Any luck so far?

I'll see if I can add a mode to get tintin working with nohup, which might fix this problem as well.
